pyproject.toml

```diff
@@ -233,10 +233,6 @@ module = 'gt4py.next.iterator.runtime'
 [tool.pytest]
 
 [tool.pytest.ini_options]
-addopts = "--instafail"
-filterwarnings = [
-  "ignore::UserWarning"
```

Comment:
Is there a better way to ignore the embedded backend warnings? They are completely useless (not only on the CI) and they locally on most terminal emulators on default settings they will push the actual failures out of the terminal history.

Reply:
If we could instead make sure that warning is only emitted max once per run, that would be preferrable